ModelParseException: SAXException while parsing input stream

**Describe the bug**
Observed as error in production logs.

**Expected behavior**
- SAXParisng exceptions are not logged as error

**Log/Stacktrace**
https://console.cloud.google.com/errors/CK7AzeKf3rSTYA?service=zeebe&time=P7D&project=camunda-cloud-240911
https://console.cloud.google.com/errors/COn7jpDXj4qlPQ?service=zeebe&time=P7D&project=camunda-cloud-240911

**Environment:**
Camunda Cloud / Prod / 0.24.2, 0.24.3

Happened again with 0.26.1:

```
org.camunda.bpm.model.xml.ModelParseException: SAXException while parsing input stream
	at org.camunda.bpm.model.xml.impl.util.DomUtil.parseInputStream(DomUtil.java:245) ~[camunda-xml-model-7.14.0.jar:7.14.0]
	at org.camunda.bpm.model.xml.impl.parser.AbstractModelParser.parseModelFromStream(AbstractModelParser.java:131) ~[camunda-xml-model-7.14.0.jar:7.14.0]
	at io.zeebe.model.bpmn.impl.BpmnParser.parseModelFromStream(BpmnParser.java:62) ~[zeebe-bpmn-model-0.26.1.jar:0.26.1]
	at io.zeebe.model.bpmn.Bpmn.doReadModelFromInputStream(Bpmn.java:390) ~[zeebe-bpmn-model-0.26.1.jar:0.26.1]
	at io.zeebe.model.bpmn.Bpmn.readModelFromStream(Bpmn.java:284) ~[zeebe-bpmn-model-0.26.1.jar:0.26.1]
	at io.zeebe.engine.processing.deployment.transform.DeploymentTransformer.readWorkflowDefinition(DeploymentTransformer.java:213) ~[zeebe-workflow-engine-0.26.1.jar:0.26.1]
	at io.zeebe.engine.processing.deployment.transform.DeploymentTransformer.transformResource(DeploymentTransformer.java:108) ~[zeebe-workflow-engine-0.26.1.jar:0.26.1]
	at io.zeebe.engine.processing.deployment.transform.DeploymentTransformer.transform(DeploymentTransformer.java:86) ~[zeebe-workflow-engine-0.26.1.jar:0.26.1]
	at io.zeebe.engine.processing.deployment.TransformingDeploymentCreateProcessor.processRecord(TransformingDeploymentCreateProcessor.java:69) ~[zeebe-workflow-engine-0.26.1.jar:0.26.1]
	at io.zeebe.engine.processing.streamprocessor.TypedRecordProcessor.processRecord(TypedRecordProcessor.java:51) ~[zeebe-workflow-engine-0.26.1.jar:0.26.1]
	at io.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.lambda$processInTransaction$2(ProcessingStateMachine.java:269) ~[zeebe-workflow-engine-0.26.1.jar:0.26.1]
	at io.zeebe.db.impl.rocksdb.transaction.ZeebeTransaction.run(ZeebeTransaction.java:79) ~[zeebe-db-0.26.1.jar:0.26.1]
	at io.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.processInTransaction(ProcessingStateMachine.java:260) ~[zeebe-workflow-engine-0.26.1.jar:0.26.1]
	at io.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.processEvent(ProcessingStateMachine.java:229) ~[zeebe-workflow-engine-0.26.1.jar:0.26.1]
	at io.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.tryToReadNextEvent(ProcessingStateMachine.java:205) ~[zeebe-workflow-engine-0.26.1.jar:0.26.1]
	at io.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.readNextEvent(ProcessingStateMachine.java:196) ~[zeebe-workflow-engine-0.26.1.jar:0.26.1]
	at io.zeebe.util.sched.ActorJob.invoke(ActorJob.java:73) [zeebe-util-0.26.1.jar:0.26.1]
	at io.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-0.26.1.jar:0.26.1]
	at io.zeebe.util.sched.ActorTask.execute(ActorTask.java:122) [zeebe-util-0.26.1.jar:0.26.1]
	at io.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:94) [zeebe-util-0.26.1.jar:0.26.1]
	at io.zeebe.util.sched.ActorThread.doWork(ActorThread.java:78) [zeebe-util-0.26.1.jar:0.26.1]
	at io.zeebe.util.sched.ActorThread.run(ActorThread.java:191) [zeebe-util-0.26.1.jar:0.26.1]
Caused by: org.xml.sax.SAXException: Fatal Error: URI=null Line=18: The prefix "zeebe" for element "zeebe:taskDefinition" is not bound.
	at org.camunda.bpm.model.xml.impl.util.DomUtil$DomErrorHandler.fatalError(DomUtil.java:207) ~[camunda-xml-model-7.14.0.jar:7.14.0]
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(Unknown Source) ~[?:?]
	at com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(Unknown Source) ~[?:?]
	at javax.xml.parsers.DocumentBuilder.parse(Unknown Source) ~[?:?]
	at org.camunda.bpm.model.xml.impl.util.DomUtil.parseInputStream(DomUtil.java:240) ~[camunda-xml-model-7.14.0.jar:7.14.0]
	... 21 more
```

--- a/engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/transform/BpmnResourceTransformer.java
+++ b/engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/transform/BpmnResourceTransformer.java
@@ -29,9 +29,9 @@ import io.camunda.zeebe.util.Either;
 import io.camunda.zeebe.util.buffer.BufferUtil;
 import java.util.Collection;
 import java.util.function.Function;
-import java.util.stream.Collectors;
 import org.agrona.DirectBuffer;
 import org.agrona.io.DirectBufferInputStream;
+import org.xml.sax.SAXException;
 
 public final class BpmnResourceTransformer implements DeploymentResourceTransformer {
 
@@ -60,24 +60,35 @@ public final class BpmnResourceTransformer implements DeploymentResourceTransfor
   @Override
   public Either<Failure, Void> transformResource(
       final DeploymentResource resource, final DeploymentRecord deployment) {
+    try {
+      final BpmnModelInstance definition = readProcessDefinition(resource);
+      final String validationError = validator.validate(definition);
 
-    final BpmnModelInstance definition = readProcessDefinition(resource);
-    final String validationError = validator.validate(definition);
+      if (validationError == null) {
+        // transform the model to avoid unexpected failures that are not covered by the validator
+        bpmnTransformer.transformDefinitions(definition);
 
-    if (validationError == null) {
-      // transform the model to avoid unexpected failures that are not covered by the validator
-      bpmnTransformer.transformDefinitions(definition);
+        return checkForDuplicateBpmnId(definition, resource, deployment)
+            .map(
+                ok -> {
+                  transformProcessResource(deployment, resource, definition);
+                  return null;
+                });
 
-      return checkForDuplicateBpmnId(definition, resource, deployment)
-          .map(
-              ok -> {
-                transformProcessResource(deployment, resource, definition);
-                return null;
-              });
-
-    } else {
-      final var failureMessage =
-          String.format("'%s': %s", resource.getResourceName(), validationError);
+      } else {
+        final var failureMessage =
+            String.format("'%s': %s", resource.getResourceName(), validationError);
+        return Either.left(new Failure(failureMessage));
+      }
+    } catch (final Exception e) {
+      final var cause = e.getCause();
+      final String failureMessage;
+      if (cause instanceof SAXException) {
+        failureMessage =
+            String.format("'%s': SAXException %s", resource.getResourceName(), cause.getMessage());
+      } else {
+        failureMessage = String.format("'%s': %s", resource.getResourceName(), e.getMessage());
+      }
       return Either.left(new Failure(failureMessage));
     }
   }
@@ -96,7 +107,7 @@ public final class BpmnResourceTransformer implements DeploymentResourceTransfor
     final var bpmnProcessIds =
         process.getDefinitions().getChildElementsByType(Process.class).stream()
             .map(BaseElement::getId)
-            .collect(Collectors.toList());
+            .toList();
 
     return record.getProcessesMetadata().stream()
         .filter(metadata -> bpmnProcessIds.contains(metadata.getBpmnProcessId()))

--- a/engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/transform/BpmnResourceTransformer.java
+++ b/engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/transform/BpmnResourceTransformer.java
@@ -31,7 +31,7 @@ import java.util.Collection;
 import java.util.function.Function;
 import org.agrona.DirectBuffer;
 import org.agrona.io.DirectBufferInputStream;
-import org.xml.sax.SAXException;
+import org.camunda.bpm.model.xml.ModelParseException;
 
 public final class BpmnResourceTransformer implements DeploymentResourceTransformer {
 
@@ -60,43 +60,44 @@ public final class BpmnResourceTransformer implements DeploymentResourceTransfor
   @Override
   public Either<Failure, Void> transformResource(
       final DeploymentResource resource, final DeploymentRecord deployment) {
-    try {
-      final BpmnModelInstance definition = readProcessDefinition(resource);
-      final String validationError = validator.validate(definition);
 
-      if (validationError == null) {
-        // transform the model to avoid unexpected failures that are not covered by the validator
-        bpmnTransformer.transformDefinitions(definition);
+    return readProcessDefinition(resource)
+        .flatMap(
+            definition -> {
+              final String validationError = validator.validate(definition);
 
-        return checkForDuplicateBpmnId(definition, resource, deployment)
-            .map(
-                ok -> {
-                  transformProcessResource(deployment, resource, definition);
-                  return null;
-                });
+              if (validationError == null) {
+                // transform the model to avoid unexpected failures that are not covered by the
+                // validator
+                bpmnTransformer.transformDefinitions(definition);
 
-      } else {
-        final var failureMessage =
-            String.format("'%s': %s", resource.getResourceName(), validationError);
-        return Either.left(new Failure(failureMessage));
-      }
-    } catch (final Exception e) {
-      final var cause = e.getCause();
-      final String failureMessage;
-      if (cause instanceof SAXException) {
-        failureMessage =
-            String.format("'%s': SAXException %s", resource.getResourceName(), cause.getMessage());
-      } else {
-        failureMessage = String.format("'%s': %s", resource.getResourceName(), e.getMessage());
-      }
-      return Either.left(new Failure(failureMessage));
-    }
+                return checkForDuplicateBpmnId(definition, resource, deployment)
+                    .map(
+                        ok -> {
+                          transformProcessResource(deployment, resource, definition);
+                          return null;
+                        });
+
+              } else {
+                final var failureMessage =
+                    String.format("'%s': %s", resource.getResourceName(), validationError);
+                return Either.left(new Failure(failureMessage));
+              }
+            });
   }
 
-  private BpmnModelInstance readProcessDefinition(final DeploymentResource deploymentResource) {
-    final DirectBuffer resource = deploymentResource.getResourceBuffer();
-    final DirectBufferInputStream resourceStream = new DirectBufferInputStream(resource);
-    return Bpmn.readModelFromStream(resourceStream);
+  private Either<Failure, BpmnModelInstance> readProcessDefinition(
+      final DeploymentResource deploymentResource) {
+    try {
+      final DirectBuffer resource = deploymentResource.getResourceBuffer();
+      final DirectBufferInputStream resourceStream = new DirectBufferInputStream(resource);
+      return Either.right(Bpmn.readModelFromStream(resourceStream));
+    } catch (final ModelParseException e) {
+      final var failureMessage =
+          String.format(
+              "'%s': %s", deploymentResource.getResourceName(), e.getCause().getMessage());
+      return Either.left(new Failure(failureMessage));
+    }
   }
 
   private Either<Failure, ?> checkForDuplicateBpmnId(

--- a/engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/DeploymentRejectionTest.java
+++ b/engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/DeploymentRejectionTest.java
@@ -229,24 +229,9 @@ public class DeploymentRejectionTest {
         .hasIntent(DeploymentIntent.CREATE)
         .hasRejectionType(RejectionType.INVALID_ARGUMENT);
 
-    /*
-     The detail rejection reason is an i18n SAXException, so here use the matches instead.
-
-     In Chinese
-     "Expected to deploy new resources, but encountered the following errors:
-     '/processes/saxexception-error-8026.bpmn': SAXException Error: URI=null Line=33:
-     cvc-complex-type.3.2.2: 元素 'bpmndi:BPMNPlane' 中不允许出现属性 'stroke'。"
-
-     In English
-     "Expected to deploy new resources, but encountered the following errors:
-     '/processes/saxexception-error-8026.bpmn': SAXException Error: URI=null Line=33:
-     cvc-complex-type.3.2.2: Attribute 'stroke' is not allowed to appear in element
-     'bpmndi:BPMNPlane'.
-    */
-
     assertThat(deploymentRejection.getRejectionReason())
         .describedAs("rejection should contain enough detail rejection reason")
-        .contains("SAXException Error")
+        .contains("saxexception-error-8026.bpmn")
         .contains("cvc-complex-type.3.2.2")
         .contains("stroke")
         .contains("bpmndi:BPMNPlane");

--- a/engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/DeploymentRejectionTest.java
+++ b/engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/DeploymentRejectionTest.java
@@ -21,9 +21,6 @@ import io.camunda.zeebe.protocol.record.RejectionType;
 import io.camunda.zeebe.protocol.record.intent.DeploymentIntent;
 import io.camunda.zeebe.protocol.record.value.DeploymentRecordValue;
 import io.camunda.zeebe.test.util.record.RecordingExporterTestWatcher;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
 import org.junit.ClassRule;
 import org.junit.Rule;
 import org.junit.Test;
@@ -52,14 +49,13 @@ public class DeploymentRejectionTest {
   }
 
   @Test
-  public void shouldRejectDeploymentIfNotValidDesignTimeAspect() throws Exception {
+  public void shouldRejectDeploymentIfNotValidDesignTimeAspect() {
     // given
-    final Path path = Paths.get(getClass().getResource("/processes/invalid_process.bpmn").toURI());
-    final byte[] resource = Files.readAllBytes(path);
+    final String resource = "/processes/invalid_process.bpmn";
 
     // when
     final Record<DeploymentRecordValue> rejectedDeployment =
-        ENGINE.deployment().withXmlResource(resource).expectRejection().deploy();
+        ENGINE.deployment().withXmlClasspathResource(resource).expectRejection().deploy();
 
     // then
     Assertions.assertThat(rejectedDeployment)
@@ -72,15 +68,13 @@ public class DeploymentRejectionTest {
   }
 
   @Test
-  public void shouldRejectDeploymentIfNotValidRuntimeAspect() throws Exception {
+  public void shouldRejectDeploymentIfNotValidRuntimeAspect() {
     // given
-    final Path path =
-        Paths.get(getClass().getResource("/processes/invalid_process_condition.bpmn").toURI());
-    final byte[] resource = Files.readAllBytes(path);
+    final String resource = "/processes/invalid_process_condition.bpmn";
 
     // when
     final Record<DeploymentRecordValue> rejectedDeployment =
-        ENGINE.deployment().withXmlResource(resource).expectRejection().deploy();
+        ENGINE.deployment().withXmlClasspathResource(resource).expectRejection().deploy();
 
     // then
     Assertions.assertThat(rejectedDeployment)
@@ -94,18 +88,16 @@ public class DeploymentRejectionTest {
   }
 
   @Test
-  public void shouldRejectDeploymentIfOneResourceIsNotValid() throws Exception {
+  public void shouldRejectDeploymentIfOneResourceIsNotValid() {
     // given
-    final Path path1 = Paths.get(getClass().getResource("/processes/invalid_process.bpmn").toURI());
-    final Path path2 = Paths.get(getClass().getResource("/processes/collaboration.bpmn").toURI());
-    final byte[] resource1 = Files.readAllBytes(path1);
-    final byte[] resource2 = Files.readAllBytes(path2);
+    final String resource1 = "/processes/invalid_process.bpmn";
+    final String resource2 = "/processes/collaboration.bpmn";
 
     final Record<DeploymentRecordValue> rejectedDeployment =
         ENGINE
             .deployment()
-            .withXmlResource(resource1)
-            .withXmlResource(resource2)
+            .withXmlClasspathResource(resource1)
+            .withXmlClasspathResource(resource2)
             .expectRejection()
             .deploy();
 
@@ -197,12 +189,11 @@ public class DeploymentRejectionTest {
     Assertions.assertThat(deploymentRejection)
         .hasRejectionType(RejectionType.INVALID_ARGUMENT)
         .hasRejectionReason(
-            "Expected to deploy new resources, but encountered the following errors:\n"
-                + "'p1.bpmn': - Element: start-event-1\n"
-                + "    - ERROR: Invalid timer cycle expression ("
-                + "failed to evaluate expression "
-                + "'INVALID_CYCLE_EXPRESSION': no variable found for name "
-                + "'INVALID_CYCLE_EXPRESSION')\n");
+            """
+                Expected to deploy new resources, but encountered the following errors:
+                'p1.bpmn': - Element: start-event-1
+                    - ERROR: Invalid timer cycle expression (failed to evaluate expression 'INVALID_CYCLE_EXPRESSION': no variable found for name 'INVALID_CYCLE_EXPRESSION')
+                """);
   }
 
   @Test
@@ -219,5 +210,45 @@ public class DeploymentRejectionTest {
     Assertions.assertThat(deploymentRejection)
         .hasRejectionType(RejectionType.EXCEEDED_BATCH_RECORD_SIZE)
         .hasRejectionReason("");
+  }
+
+  // https://github.com/camunda/zeebe/issues/8026
+  @Test
+  public void shouldRejectDeploymentOfSAXException() {
+    // given
+    final String resource = "/processes/saxexception-error-8026.bpmn";
+
+    // when
+    final Record<DeploymentRecordValue> deploymentRejection =
+        ENGINE.deployment().withXmlClasspathResource(resource).expectRejection().deploy();
+
+    // then
+    Assertions.assertThat(deploymentRejection)
+        .hasKey(ExecuteCommandResponseDecoder.keyNullValue())
+        .hasRecordType(RecordType.COMMAND_REJECTION)
+        .hasIntent(DeploymentIntent.CREATE)
+        .hasRejectionType(RejectionType.INVALID_ARGUMENT);
+
+    /*
+     The detail rejection reason is an i18n SAXException, so here use the matches instead.
+
+     In Chinese
+     "Expected to deploy new resources, but encountered the following errors:
+     '/processes/saxexception-error-8026.bpmn': SAXException Error: URI=null Line=33:
+     cvc-complex-type.3.2.2: 元素 'bpmndi:BPMNPlane' 中不允许出现属性 'stroke'。"
+
+     In English
+     "Expected to deploy new resources, but encountered the following errors:
+     '/processes/saxexception-error-8026.bpmn': SAXException Error: URI=null Line=33:
+     cvc-complex-type.3.2.2: Attribute 'stroke' is not allowed to appear in element
+     'bpmndi:BPMNPlane'.
+    */
+
+    assertThat(deploymentRejection.getRejectionReason())
+        .describedAs("rejection should contain enough detail rejection reason")
+        .contains("SAXException Error")
+        .contains("cvc-complex-type.3.2.2")
+        .contains("stroke")
+        .contains("bpmndi:BPMNPlane");
   }
 }

--- a/engine/src/test/resources/processes/saxexception-error-8026.bpmn
+++ b/engine/src/test/resources/processes/saxexception-error-8026.bpmn
@@ -1,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:bioc="http://bpmn.io/schema/bpmn/biocolor/1.0" id="Definitions_0a084vr" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="4.11.0">
+  <bpmn:process id="invalid" name="invalid process" isExecutable="true">
+    <bpmn:startEvent id="start" name="start">
+      <bpmn:outgoing>Flow_1p61drr</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:endEvent id="end" name="end">
+      <bpmn:incoming>Flow_1p61drr</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_1p61drr" sourceRef="start" targetRef="end" />
+  </bpmn:process>
+  <bpmn:message id="Message_09k35jn" name="vantage-transcode-completed">
+    <bpmn:extensionElements>
+      <zeebe:subscription correlationKey="=zeebeProcessId" />
+    </bpmn:extensionElements>
+  </bpmn:message>
+  <bpmn:message id="Message_0kumuuf" name="vantage-transcode-failed">
+    <bpmn:extensionElements>
+      <zeebe:subscription correlationKey="=zeebeProcessId" />
+    </bpmn:extensionElements>
+  </bpmn:message>
+  <bpmn:message id="Message_1vf3iyi" name="vantage-transcode-status-update">
+    <bpmn:extensionElements>
+      <zeebe:subscription correlationKey="=zeebeProcessId" />
+    </bpmn:extensionElements>
+  </bpmn:message>
+  <bpmn:message id="Message_0afsp8q" name="vantage-transcode-progress-update">
+    <bpmn:extensionElements>
+      <zeebe:subscription correlationKey="=zeebeProcessId" />
+    </bpmn:extensionElements>
+  </bpmn:message>
+  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
+    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="invalid" stroke="#000" fill="#fff">
+      <bpmndi:BPMNEdge id="Flow_1p61drr_di" bpmnElement="Flow_1p61drr" bioc:stroke="#000" bioc:fill="#fff">
+        <di:waypoint x="208" y="100" />
+        <di:waypoint x="302" y="100" />
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="start" bioc:stroke="#000" bioc:fill="#fff">
+        <dc:Bounds x="172" y="82" width="36" height="36" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="182" y="125" width="22" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Event_0mhoyjr_di" bpmnElement="end" bioc:stroke="#000" bioc:fill="#fff">
+        <dc:Bounds x="302" y="82" width="36" height="36" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="311" y="125" width="19" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+    </bpmndi:BPMNPlane>
+  </bpmndi:BPMNDiagram>
+</bpmn:definitions>

